### DISCUSSION TITLE
Profile config location?

### DISCUSSION THREAD

**Discussion by Wright4i** [codefori/vscode-ibmi#801] [Q&A] [answered]
Is there a json file for Code for IBM i Profiles?

I'd like to quickly populate multiple library lists across different profiles without doing it all by hand.

**Comment by Wright4i** [comment, marked as answer]
Nevermind. Found it - settings.json "connectionProfiles"

For windows:
%appdata%\Code\User\settings.json

**Comment by worksofliam** [comment]
Profiles are stored as JSON in the VS Code settings. Does that count? :)

**Reply by Wright4i** [reply]
Yes it absolutely does count. For some reason I assumed profiles wouldn't be in the main settings.json

**Reply by worksofliam** [reply]
@Wright4i I do actually hate that they're in the settings, but it's just easier to put them there since they're tied to connections.
